### DISCUSSION TITLE
RadioSelect as rating stars

### DISCUSSION THREAD

**Discussion by amikphoto** [jrief/django-formset#108] [Ideas]
Could you help me to make rating stars in collection forms. I can set RadioSelect wiget for CharField model. But I can't set attrs for RadioSelect widget. How can I make stars rating selector in my forms with django formset?
Thank you very much

**Comment by jrief** [comment]
I would create a special widget for that use case, say `RatingWidget`. It should be used together with a `ChoiceField` with 5 options. Then create a web component which implements this. That new widget should then use that new web component.

Use this [Shoelace](https://shoelace.style/components/rating) component as blueprint.

Anyway, if you're patient, I will implement these components one-by-one in the next versions.
